### PR TITLE
Renewal item_db update

### PR DESCRIPTION
**Pull request by zackdreaver**
- Add new costumes
- Fix item script

Signed-off-by: zackdreaver <zackdreaver@gmail.com>

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -11567,10 +11571,12 @@
 31086,C_Long_Pony_Purple,Costume Long Pony Purple,4,0,,0,,,,0,0xFFFFFFFF,63,2,1024,,1,0,1526,{},{},{}
 31087,C_Dwarf_Beard,Costume Dwarf Beard,4,0,,0,,,,0,0xFFFFFFFF,63,2,4096,,1,0,1349,{},{},{}
 31088,C_Mad_Hatter,Costume Mad Hatter,4,0,,0,,,,0,0xFFFFFFFF,63,2,1024,,1,0,1421,{},{},{}
-31089,C_Exploding_Crimson_Flame,Costume Exploding Crimson Flame,4,0,,0,,,,0,0xFFFFFFFF,63,2,2048,,1,0,1500,{},{},{}
+31089,C_Exploding_Crimson_Flame,Costume Exploding Crimson Flame,4,0,,0,,,,0,0xFFFFFFFF,63,2,2048,,1,0,0,{},{},{}
```
Comment (secretdataz) [inline db/re/item_db.txt]: Why has this item's viewid become 0?

Comment (zackdreaver) [inline db/re/item_db.txt]: It need a script like costume wordstar/musical decoration.
There is no viewid for this one.

**Comment by secretdataz** [comment]
@zackdreaver Please reply when you're done with this PR. Thanks

**Comment by zackdreaver** [comment]
I'm done with the commits, waiting for review.

**Comment by cydh** [comment]
@zackdreaver bro, I would appreciate if you write your references/sources as link or just simple comment for this update

**Pull request merged**